src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-put.test.ts 

```diff
@@ -0,0 +1,50 @@
+import { GitHubServerApp } from "models/github-server-app";
+
+describe("PUT /jira/connect/enterprise/app/:uuid", () => {
+	const gitHubBaseUrl = "http://myinternalinstance.com";
+	const installationId = 42;
+	const uuid = "c97806fc-c433-4ad5-b569-bf5191590be2";
+	const clientIdWithTypo = "lvl.1n23j12389wndd";
+	const originalWebhookSecret = "anothersecret";
+	const originalPrivateKey = "privatekey";
+	let gitHubServerApp;
+
+	beforeEach(async () => {
+		gitHubServerApp = await GitHubServerApp.create({
+			uuid,
+			appId: 1,
+			gitHubAppName: "my awesome app",
+			gitHubBaseUrl,
+			gitHubClientId: clientIdWithTypo,
+			gitHubClientSecret: "secret",
+			webhookSecret: "anothersecret",
+			privateKey: "privatekey",
+			installationId
+		});
+	});
+
+	it("should update GitHub app when uuid is found", async () => {
+		const originalApp = await GitHubServerApp.findForUuid(uuid);
+		expect(originalApp && originalApp.gitHubClientId).toEqual(clientIdWithTypo);
+		expect(originalApp && originalApp.webhookSecret).toEqual(`encrypted:${originalWebhookSecret}`);
+		expect(originalApp && originalApp.privateKey).toEqual(`encrypted:${originalPrivateKey}`);
+
+		await GitHubServerApp.updateGitHubApp({
```

Comment:
This test is a bit strange? From the file name, it is for the `jira-connect-enterprise-app-put` handler, but the test itself is directly testing the db models. 

If it is testing the db models, I'd say change the file name to `models/github-server-app.test.ts`, if testing the handler, I'd say use the `supertest` to hit the api.